src/bsp/src/stm32f4/hal/stm32f429zi/Core/Inc/hivemind_hal.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __HIVEMIND_HAL_H__
+#define __HIVEMIND_HAL_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "gpio.h"
+#include "main.h"
+#include "stm32f4xx_hal_conf.h"
+#include "usart.h"
+
+#define huart_print (&huart3)
```

Comment:
I can't find it in the LLVM coding standard, but by experience, shouldn't constants be all caps?